docs/UserGuide.md

```diff
@@ -152,6 +154,7 @@ Examples:
 ### Deleting a restaurant : `delete`
 
 Deletes the specified restaurant from the address book.
+Note: Any restaurant filters will be reset upon running this command.
```

Comment:
this the tags?

Reply:
find or price

Reply:
we have predicates for names, tags and price, these are our filters